UML-Diagram/ExceptionErrorHandling.puml

```diff
@@ -0,0 +1,44 @@
+@startuml
+
+package "Exception Handling" {
+    enum Severity {
+        {static} + LOW
+        {static} + MEDIUM
+        {static} + HIGH
+        {static} + CRITICAL
+    }
+
+    abstract class GlobalException {
+        + Message : String
+        + Severity : Severity
+        + GlobalException(String message, Severity severity)
+    }
+
+    class BookNotFoundException {
+        + BookNotFoundException(String message)
+    }
+
+    class PatronNotFoundException {
+        + PatronNotFoundException(String message)
+    }
+
```

Comment:
its a system for manage the Errors i think that we need more Exceptions, such as BorrowNotFoundException NullBorrowException, and others.

Reply:
You're right. We'll need more but now it's a little complicate to control or implement for sure in those places. This is a suggest of how we will control it and base on it, on the code I'll made more implementation according to what it's needed and once it's done, I'll update completely again the UML for this.

Reply:
You're right. We'll need more but now it's a little complicate to control or implement for sure in those places. I mean it will take me more time to review completely the UML and put the exceptions. This is a suggest or way of how we will control it. And base on it, in the code I'll do more implementations according to what it's needed and once it's done, I'll update completely again the UML for this.

At this point, I had implemented on the classes where it's needed, using of LogService and ExceptionHandling.